pkg/gofr/handler.go

```diff
@@ -150,3 +147,18 @@ func panicRecoveryHandler(re any, log logging.Logger, panicked chan struct{}) {
 		StackTrace: string(debug.Stack()),
 	})
 }
+
+// Log the error(if any) with traceID and errorMessage.
+func (h handler) logError(traceID string, err error) {
+	if err != nil {
+		errorLog := &ErrorLogEntry{TraceID: traceID, Error: err.Error()}
+		h.container.Logger.Error(errorLog)
+	}
```

Comment:
```suggestion
	if err != nil {
		return
	}
	errorLog := &ErrorLogEntry{TraceID: traceID, Error: err.Error()}
	h.container.Logger.Error(errorLog)
```

Reply:
I'd just extracted the existing code to a new function, also isn't the logic wrong here, the error should get logged when it is not nil right?

Reply:
Indeed, I edited my post it should be OK now